Description: Do not use intersphinx plugin (which does network access)
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-07-12

--- python-requests-mock-0.4.0.orig/docs/conf.py
+++ python-requests-mock-0.4.0/docs/conf.py
@@ -40,8 +40,7 @@ import requests_mock  # noqa
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.doctest',
-              'sphinx.ext.viewcode',
-              'sphinx.ext.intersphinx']
+              'sphinx.ext.viewcode']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
